src/GitVersionExe/HelpWriter.cs

```diff
@@ -53,6 +53,9 @@ public void WriteTo(Action<string> writeAction)
  # AssemblyInfo updating
     /updateassemblyinfo
                     Will recursively search for all 'AssemblyInfo.cs' files in the git repo and update them
+    /targetprojectfiles
+                    Will target project files (cs/fs/vbproj files) to update instead of AssemblyInfo class files,
```

Comment:
Can we call the argument `updateprojectfiles` to be consistent with the rest of the `update` arguments?

```suggestion
    /updateprojectfiles
                    Will recursively search for `.csproj`, `.vbproj` and `.fsproj` files in the Git repository and update them.
```